### ISSUE TITLE
[C4GT] Network-Membership: Add tests for `MaxMembersExceededForTheBlock`

### ISSUE THREAD

**Issue by vatsa287** [dhiway/cord#293]
## Description
Subtask under : https://github.com/cord-network/community/issues/7

New testcase can be tested under `cargo test -p pallet-network-membership` after adding it in the code.

## Goals

- [ ] Add tests for `MaxMembersExceededForTheBlock` for pallet/network-membership

## Expected Outcome
- [ ] Test should assert for `MaxMembersExceededForTheBlock` being returned properly in all the possible calls.
- [ ] If there are more than one function returning this error code, all of them should be validated either as one test case or as multiple test cases.
       
## Acceptance Criteria
NA

## Implementation Details
Look at other test cases, and add a test case for the same.
Ex: `test_duplicate_member_request` which validates error code `MembershipAlreadyAcquired`

## Mockups / Wireframes
NA

---- 

## Product Name
CORD

## Organization Name
Dhiway

## Domain
Blockchain

## Tech Skills Needed
Rust

## Mentor(s)
@amarts 

## Complexity
[Low]

## Category
[Test]

## Sub Category
[Beginner friendly]

**Comment by Aloneking789** [comment]
assign it to me i am making a pull request please merge it

**Comment by PriyaD17** [comment]
hey @amarts in this project we have to assert for Maximum Members Exceeded For The Block and we need to add this test to pallets/network-membership path that should pass the Cargo test. Have I got the right idea? let me know if I am wrong otherwise please assign me this issue so I can start working on it.
Thankyou

**Comment by vatsa287** [comment]
@PriyaD17 
Yes, the storage map has compile time upper bound set to `type MaxMembersPerBlock: Get<u32>;` for the storage map `MembershipsExpiresOn`. Try adding more than this number for the error to hit.

**Comment by VedantKhairnar** [comment]
Hello @PriyaD17 @Aloneking789 
Pls directly raise the PR.
Thanks.

@vatsa287 Pls assign it if possible.

**Comment by Vaibhavsahu2810** [comment]
Hello @vatsa287 , I would like to work on this issue it seems inactive for a long time

**Comment by priyankeshh** [comment]
@vatsa287  I saw that still no PR has been made for this issue, I would like to work on it

**Comment by devmuse7** [comment]
Hi @vatsa287  I would like to take this issue.
I come with years of experience and hands on experience on open source projects.
You can check my Github Profile.
